sal_interfaces/MTRotator/MTRotator_Telemetry.xml

```diff
@@ -110,20 +117,34 @@
   <SALTelemetry>
     <Subsystem>MTRotator</Subsystem>
     <EFDB_Topic>MTRotator_motors</EFDB_Topic>
-    <Description>Encoder readings from each rotator motor</Description>
+    <Description>Data about the two rotator motors, in order A, B</Description>
     <item>
       <EFDB_Name>calibrated</EFDB_Name>
-      <Description>Encoder readings scaled to degrees of motion of the rotator.</Description>
+      <Description>Motor encoder readings, scaled to degrees of motion of the rotator.</Description>
       <IDL_Type>double</IDL_Type>
       <Units>deg</Units>
       <Count>2</Count>
     </item>
     <item>
       <EFDB_Name>raw</EFDB_Name>
-      <Description>Encoder readings in raw counts.</Description>
+      <Description>Motor encoder readings in raw counts.</Description>
       <IDL_Type>double</IDL_Type>
       <Units>unitless</Units>
       <Count>2</Count>
     </item>
+    <item>
+      <EFDB_Name>current</EFDB_Name>
+      <Description>Motor current. Initially (2021-08-17) this is an estimate from the Simulink plant model.</Description>
```

Comment:
Since there is no C code of Simulink plant model in the low-level controller, I think the values will be 0 as the default.